.travis/unittest.sh

```diff
@@ -24,6 +24,6 @@ unittest(){
 trap 'abort' 0
 set -e
 
-unittest .
+# unittest .
```

Comment:
The unit test in travis-ci is disabled because it requires the installation of caffe2, as the backend of ONNX.

Reply:
Yes, seems caffe2 backend is necessary, let me try to add the dependencies in the CI.

Reply:
@pkuyym the installation of caffe2 always failed in local machine, and seems that it is very sensitive to the dependencies, e.g. version of `cmake`, `protobuf` etc. You can find if there is a robust way, so that we can integrate it in CI.